packages/dropdowns/demo/menu.stories.mdx

```diff
@@ -56,7 +56,8 @@ import { MENU_ITEMS as ITEMS } from './stories/data';
       items: ITEMS,
       selectedItems: [ITEMS[5]],
       hasMedia: false,
-      isOpen: false
+      isOpen: false,
+      appendToNode: 'undefined'
```

Comment:
This and the following addition to `argTypes` tends to muddy the default API definition and isn't strictly in line with the repo's [demo documentation](https://github.com/zendeskgarden/react-components/blob/main/docs/demo.md) guidelines. I might suggest removing these from the `Menu` story and instead adding a one-off [[patterns]](https://github.com/zendeskgarden/react-components/blob/main/docs/demo.md#patterns) story that fully illustrates what you're going for with this change. See other `[patterns]` for reference:
- https://zendeskgarden.github.io/react-components/?path=/story/packages-avatars-patterns--chrome
- https://zendeskgarden.github.io/react-components/?path=/story/packages-forms-patterns--file-upload
- https://zendeskgarden.github.io/react-components/?path=/story/packages-tags-patterns--faux-input

Reply:
No worries, that `~patterns` story is created now.

I have also removed it from the main `Menu` story. With that said, I myself would find that arg handy if I was doing a11y testing, and I wanted to test all of the different permutations of the menu. It's an available prop for the `Menu` component, so it makes sense to document it in the story, at least from my experience. But up to you.